automation/upload_tls.py

```diff
@@ -93,32 +103,55 @@ def main():
     # Add Auth Header by default to all requests.
     REQUEST.headers.update({'Authorization': f'Bearer {access_token}'})
 
-    # Retrieve all the keystore
-    keystores_list = apigee_tls_keystore.get_keystores_list(REQUEST, org_name, env_name)
+    if env_name != 'portal':
```

Comment:
why this if statment? we still need to get all keystores if environment was portal

Reply:
We get them again for portal env in the code below, this if is to be sure that update reference will not be executed when env is portal.